auditlog/management/commands/auditlogflush.py

```diff
@@ -42,11 +56,45 @@ def handle(self, *args, **options):
             )
             answer = response == "y"
 
-        if answer:
+        if not answer:
+            self.stdout.write("Aborted.")
+            return
+
+        if not truncate:
             entries = LogEntry.objects.all()
             if before is not None:
                 entries = entries.filter(timestamp__date__lt=before)
             count, _ = entries.delete()
             self.stdout.write("Deleted %d objects." % count)
+            return
```

Comment:
I think this line here is useless because we don't have any code after the `if - else` block.

Reply:
It got fixed, thanks for suggestion.